src/server.c

```diff
@@ -87,16 +87,12 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
 void get_d20(int fd)
 {
     // Generate a random number between 1 and 20 inclusive
-    
-    ///////////////////
-    // IMPLEMENT ME! //
-    ///////////////////
+    int num = rand() % 20 + 1;
+    char str[12];
+    sprintf(str, "<h1>%d</h1>", num);
```

Comment:
very nice! you don't _need_ the `<h1>` tags, but I also included them - makes it look much nicer on the browser.
looks perfect. didn't even waste memory space on line 91